src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2190,4 +2202,15 @@ PeerImp::getScore (bool haveItem)
    return score;
 }
 
+bool
+PeerImp::highLatency ()
```

Comment:
I'm not so sure about the value of `const` on virtual members for objects that maintain abstract interfaces (like `class Peer`).